apps/lib/apps.c

```diff
@@ -806,6 +822,9 @@ int load_key_certs_crls(const char *uri, int maybe_stdin,
 
  end:
     OSSL_STORE_close(ctx);
+    if (failed != NULL)
+        return 0;
```

Comment:
This no longer calls ERR_print_errors() if it returns here

Reply:
> This no longer calls ERR_print_errors() if it returns here

Good point, and this was missing also in case of `return 0` on out-of-memory. Fixed.